tools/gradle/pmd/rules.xml

```diff
@@ -10,6 +10,7 @@
     <exclude-pattern>.*/generated/.*</exclude-pattern>
     <exclude-pattern>.*/airbyte-integrations/.*</exclude-pattern>
     <exclude-pattern>.*/test-integration/.*</exclude-pattern>
+    <exclude-pattern>.*/airbyte-cdk/.*</exclude-pattern>
```

Comment:
Carried over from previously being ignored under the `airbyte-integrations` filter

Reply:
I'm under the impression that pmd was dead in this repo and effectively a holdover from before airbyte-platform was spun off.

Reply:
Pmd was causing failures during the Gradle `build` task - so while we perhaps sunsetted its usage, it doesn't look like the plugin or its tasks are fully disabled. (Unless I accidentally pulled them back in somehow.)